package-lock.json

```diff
@@ -126,9 +126,9 @@
 			}
```

Comment:
Can you run another update on this file? I still see some changes when I run `npm install` on my machine. (typescript changed from 2.9.2 to 3.2.2)